src/handlers/transfers/FxFulfilService.js

```diff
@@ -325,20 +434,28 @@ class FxFulfilService {
   }
 
   async kafkaProceed(kafkaOpts) {
-    return this.Kafka.proceed(
-      this.Config.KAFKA_CONFIG,
-      this.params,
-      kafkaOpts
-    )
+    return this.Kafka.proceed(this.Config.KAFKA_CONFIG, this.params, {
+      hubName: this.Config.HUB_NAME,
+      ...kafkaOpts
```

Comment:
shouldn't it be like this:
```
... , {
  ...kafkaOpts,
  hubName: this.Config.HUB_NAME
})  
```
to avoid possible overriding of `hubName` with a value from `kafkaOpts`?